src/Microsoft.Data.SqlClient/src/Interop/Windows/Handles/DebugSafeHandle.netcore.cs

```diff
@@ -2,16 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#if DEBUG && !NET8_0_OR_GREATER
+#if DEBUG && !NETFRAMEWORK && !NET8_0_OR_GREATER
```

Comment:
```suggestion
#if DEBUG && NET6_0
```

Reply:
I think it communicates the intent a bit better, but up to you. We'll delete them soon anyway.

Reply:
My rationale for using !NET8_0_OR_GREATER is because covers all versions below net8. I know *we* only build net6 and net8, and I guess it would be silly for someone to pull down the source and try to build it for eg, net7, but this is a fairly benign way to cover more bases. 

I think of it similar to how we could do `string.IndexOf("foo") == -1` or we could do `string.IndexOf("foo") < 0`, and how the latter is generally thought to be safer.